docs/sphinx/_toc.yml.in

```diff
@@ -15,6 +15,7 @@ subtrees:
 
 - caption: Reference
   entries:
+<<<<<<< HEAD
```

Comment:
TOC needs to be resolved.

Reply:
Thanks, resolved.

Reply:
It is an occuring issue. I do not really know the reason, but I have an intuition that something is cashed and it makes a conflict. I wanted to open a ticket to investigate, but I forgot it. It happens with the original version too. I think I should take a look onto the CI policy, but that is just my first impression.

I did not modify anything just I pushed the commit with a new hash number and the CI works.